code/modules/antagonists/heretic/magic/moon_smile.dm

```diff
@@ -28,7 +28,8 @@
 /datum/action/cooldown/spell/pointed/moon_smile/cast(mob/living/carbon/human/cast_on)
 	. = ..()
 	/// The duration of these effects are based on sanity, mainly for flavor but also to make it a weaker alpha strike
-	var/moon_smile_duration = (150 - cast_on.mob_mood.sanity) / 10
+	var/maximum_duration = 15 SECONDS
+	var/moon_smile_duration = ((SANITY_MAXIMUM - cast_on.mob_mood.sanity) / (SANITY_MAXIMUM - SANITY_INSANE)) * maximum_duration
```

Comment:
```suggestion
	var/moon_smile_duration = ((SANITY_MAXIMUM - cast_on.mob_mood.sanity) / SANITY_MAXIMUM) * maximum_duration
```
the SANITY_INSANE value is 0 anyway.

Reply:
I know, but just in case it ever changes, I use the define so that the scaling stays accurate.